README.md

```diff
@@ -88,3 +90,35 @@ require('cmake').setup({
   }
 })
 ```
+
+### Statusline integration example
+
+```lua
+g_line = ""  -- can be displayed in statusline, updated in build_on_output
```

Comment:
I prefer to not prefix global variables. Also how about to rename it into `progress`?